test/gw-ci/CMakeLists.txt

```diff
@@ -99,40 +381,31 @@ if (WORKFLOW_TESTS)
     # "gdas_marineanlchkpt"
     # "gdas_marineanlfinal"
     )
-  add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
+  add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${WORKING_DIRECTORY} ${PROJECT_SOURCE_DIR} "${HALF_CYCLE_TASKS}" "${FULL_CYCLE_TASKS}")
   set(letkf FALSE)
-endif()
-
-option(RUN_GW_CI "Enable the global-workflow CI tests" OFF)
-if (RUN_GW_CI)
-  # Aero-Land DA, C96
-  # -----------------
-  set(pslot "Aero-Snow-3DVAR-C96")
-  set(YAML_PATH ${HOMEgfs}/ci/cases/pr/C96C48_hybatmaerosnowDA.yaml)
-  set(TASK_LIST)  # empty list for now
-  add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
-
-  # Atm DA, C96/C48
-  # ---------------
-  set(pslot "Atm-hyb-C96C48")
-  set(YAML_PATH ${HOMEgfs}/ci/cases/pr/C96C48_ufs_hybatmDA.yaml)
-  set(TASK_LIST)  # empty list for now
-  add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
 
   # GFSv17, 3DVAR prototype
   # -----------------------
-  set(pslot "GFSv17-3DVAR-C384mx025")
-  set(YAML_PATH ${HOMEgfs}/ci/cases/gfsv17/C384mx025_3DVarAOWCDA.yaml)
-  set(TASK_LIST
-    "gdas_prepoceanobs"
-    "gdas_marinebmat"
-    "gdas_marineanlinit"
-    "gdas_marineanlvar"
-    "gdas_marineanlchkpt"
-    "gdas_marineanlfinal"
-    "gdas_ocnanalvrfy"
-    "gdas_prep"
-    "gdas_anal"
-    )
-  add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
+  option(TEST_GFS17 "Enable the GFSv17 prototype CI tests" OFF)
+  if(TEST_GFS17)
+    set(pslot "GFSv17-3DVAR-C384mx025")
+    set(YAML_PATH ${HOMEgfs}/ci/cases/gfsv17/${pslot}.yaml)
```

Comment:
I am trying to run this set of tests, and it can't find the `GFSv17-3DVAR-C384mx025.yaml` in `global-workflow/ci/cases/gfsv17`. I don't see this file on develop, and I don't see it in https://github.com/NOAA-EMC/global-workflow/pull/3120. What am I missing? (I have not yet tried to run with https://github.com/NOAA-EMC/global-workflow/pull/3120)

Reply:
@shlyaeva Yeah, that was a mistake on my part. I assumed the GFSv17 test name had the same naming convention as the low resolution marine CI. Thanks for pointing this out, I'll make a bugfix PR tomorrow morning.

Reply:
Oh good, thank you for the clarification! I am still figuring out how to run things, so I wasn't sure whether I missed something.